po/ja.po

```diff
@@ -1373,36 +1377,43 @@ msgid "**In-Person Event Checklist (Carpentries Workshops, CarpentryCon, Carpent
 "* Listen to the reporter and write down as many details as possible.\n"
 "* Ask the reporter to report the incident via the [Code of Conduct Incident Report Form][reporting-form]. If they would rather you complete the report, complete the report with their assistance. \n"
 "* Inform the workshop host that there was an incident and that a report was submitted via the [incident report form][reporting-form]. If the incident involves the workshop host, report the incident directly to the Code of Conduct committee via the incident report form. "
-msgstr ""
+msgstr "**対面イベントチェックリスト (カーペントリーズのワークショップ, CarpentryCon, CarpentryConnect, カーペントリーズのインストラクターのトレーニング)**\n"
+"* 参加者の安全を確保します。\n"
+"* そうでない場合は、即時対応チェックリストを参照してください。\n"
+"* レポーターの話を聞き、できるだけ多くの詳細を書き留めてください。\n"
+"* [行動規範インシデントレポートフォーム][reporting-form]を介してインシデントをレポートするようにレポーターに依頼してください。レポーターがあなたにレポートを完成させてほしいと望む場合、レポーターの助けを借りてレポートを完成させてください。\n"
```

Comment:
Let's try to avoid using katana-ized versions of the English word if appropriate Japanese words are available. I think that will be easier to understand.
```suggestion
"* [行動規範インシデントレポートフォーム][reporting-form]を介してインシデントを報告するようにレポーター（被害者）に依頼してください。レポーター（被害者）があなたにレポートを完成させてほしいと望む場合、レポーター（被害者）の助けを借りてレポートを完成させてください。\n"
```